### PR TITLE
Rollback of #3531

### PR DESCRIPTION
**Pull request by fonnesbeck**
Going back to old tqdm until we can make the update more seamless. 

Even after installing and configuring ipywidgets, it looks like this for me:

Screenshot 2019-06-29 at 3 50 08 PM

This PR rolls back #3531

### CODE REVIEW

File: pymc3/stats.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from scipy.stats import dirichlet
 from scipy.optimize import minimize
 from scipy.signal import fftconvolve
-from tqdm.auto import tqdm
+import tqdm
```
Comment (ColCarroll) [inline pymc3/stats.py]:
```suggestion
from tqdm import tqdm
```

File: pymc3/tuning/starting.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 from scipy.optimize import minimize
 import numpy as np
 from numpy import isfinite, nan_to_num
-from tqdm.auto import tqdm
+import tqdm
```
Comment (ColCarroll) [inline pymc3/tuning/starting.py]:
```suggestion
from tqdm import tqdm
```

**Review by ColCarroll — COMMENTED** [review]
i think two more little changes

**Comment by junpenglao** [comment]
cc @vnherdeiro

**Comment by vnherdeiro** [comment]
@fonnesbeck Is this happening with a notebook inside jupyter lab or straight from jupyter notebook?

**Comment by ColCarroll** [comment]
I had similar problems - for me it was inside jupyter lab. Took a bit of updating the server and the kernel to get everything working (first I was missing `IntProgress` which threw an exception, then I got to the above, printing out what looks like an HTML repr, then got it working).

I think a nicer progressbar is a good idea, but it has to be pretty seamless to use.

**Comment by fonnesbeck** [comment]
Yes, it was jupyterlab for me as well. I don't use classic notebooks much  anymore.

**Comment by vnherdeiro** [comment]
Indeed I manage to reproduce it in jupyterlab but not in native jupyter notebooks. Quite a pity that it is so sensitive on the environment set up.

**Comment by junpenglao** [comment]
It might be worth to file an issue to tqdm re differences in jupyterlab/jupyter notebook.
Nonetheless, @vnherdeiro thanks for the effort to make progressbar better - if you find another solution please feel free to send another PR :-)

**Pull request merged**